pkg/clients/eks/nodegroup.go

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
In `clients` package, we have always had 1 package per 1 managed resource but in this case it seems `nodegroup` functions are in the `eks` package together with `cluster`. What do you think about separating them and having `nodegroup` and `cluster` separate?

Reply:
They are using the same client so it seems redundant. Do you see any advantage to separating them?

Reply:
No, not much advantage given they are using the exact same client.